tkltest/generate/unit/augment.py

```diff
@@ -290,9 +293,9 @@ def __compute_coverage_efficiency(test_dir, build_file, build_type, report_dir,
 def __initialize_test_directory(ctd_test_dir, source_test_dir):
     """Clears CTD test directory and adds test classes from the given source test directory to the CTD test directory"""
     # clear the target (ctd) directory
-    target_test_dir = ctd_test_dir + os.sep + 'monolithic'
-    shutil.rmtree(target_test_dir, ignore_errors=True)
-    os.makedirs(target_test_dir)
+    for target_test_dir in [dir for dir in os.listdir(ctd_test_dir) if os.path.isdir(dir) and dir != 'build' and dir != 'target']:
```

Comment:
At this point are we still running in monolithic? if not, then the check for `build` and `target` is redundant, because they will not be inside the tests directory

Reply:
reminder - In this PR, build directories are in the test directory.
whey will be removed only an the next PR